handlers/holiday-stop-api/src/main/scala/com/gu/holiday_stops/Handler.scala

```diff
@@ -416,6 +434,10 @@ object Handler extends Logging {
         .toApiGatewayOp(s"get subscription ${requestBody.subscriptionName}")
       account <- getAccount(accessToken, subscription.accountNumber)
         .toApiGatewayOp(s"get account ${subscription.accountNumber}")
+      billingPreview <- getBillingPreview
+        .getBillingPreview(accessToken, subscription.accountNumber, today.plusMonths(13))
+        .toApiGatewayOp(s"get account ${subscription.accountNumber}")
```

Comment:
good spot!  even better is:
```suggestion
        .toApiGatewayOp(s"get billing preview for account ${subscription.accountNumber}")
```